workflow-templates/add-depr-ticket-to-depr-board.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow runs when a ticket is created. It checks if the ticket
```

Comment:
Instead of making this a workflow template, I suggest making this a [re-usable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows) and then making a workflow template that references the re-usable workflow, this will make updating the workflow a lot easier in the future.

We started doing this with commitlint and want to move more things to that model for ease of updating.

Reusable Workflow: https://github.com/openedx/.github/blob/master/.github/workflows/commitlint.yml
Template that references that workflow: https://github.com/openedx/.github/blob/master/workflow-templates/commitlint.yml